checkov/terraform/checks/resource/aws/RDSDeletionProtection.py

```diff
@@ -7,7 +7,7 @@ class RDSDeletionProtection(BaseResourceValueCheck):
     def __init__(self):
         name = "Ensure that RDS clusters have deletion protection enabled"
         id = "CKV_AWS_139"
-        supported_resources = ['aws_rds_cluster']
+        supported_resources = ['aws_rds_cluster' ,'aws_db_instance']
```

Comment:
@JamesWoolfenden what do you think? looking at the name of the check we would either need to change/extend the name or create a separate check for it.

Reply:
id go with a separate check one for db instance, we went that way for encryption and CMK checks and it lets have the control on suppression.

Reply:
a new check for 'aws_db_instance' and new test cases and test for that new check